src/script/src/python/ffi_types/pair_tests/sample_testcases.rs

```diff
@@ -56,6 +57,34 @@ def boolean_array() -> vector[f64]:
     from greptime import vector
     from greptime import query, dataframe
     
+    try: 
+        print("query()=", query())
+    except KeyError as e:
+        print("query()=", e)
+    try: 
+        print("dataframe()=", dataframe())
+    except KeyError as e:
+        print("dataframe()=", e)
+
+    v = vector([1.0, 2.0, 3.0])
+    # This returns a vector([2.0])
+    return v[(v > 1) & (v < 3)]
+"#
+            .to_string(),
+            expect: Some(ronish!("value": vector!(Float64Vector, [2.0f64]))),
+        },
+        CoprTestCase {
+            script: r#"
+@copr(returns=["value"], backend="rspy")
+def boolean_array() -> vector[f64]:
+    from greptime import vector
+    from greptime import query, dataframe, PyDataFrame
+    
+    df = PyDataFrame.from_sql("select number from numbers limit 5")
```

Comment:
can we add a case that combine SQL and dataframe API? like 
```python
df = PyDataFrame.from_sql("blabla").filter("blabla")
```
It should be supported now.

Reply:
added something like this:
```python
df = PyDataFrame.from_sql("select number from numbers limit 5").filter(col("number") > 2)
collected = df.collect()
assert len(collected[0][0]) == 2
```

> can we add a case that combine SQL and dataframe API? like
> 
> ```python
> df = PyDataFrame.from_sql("blabla").filter("blabla")
> ```
> 
> It should be supported now.